app/models/ability.rb

```diff
@@ -62,4 +66,10 @@ def custom_permissions
     def curation_concerns
       Hyrax.config.curation_concerns
     end
+
+    def download_file_permission
+      can :read, ActiveFedora::File do |file|
+        file.mime_type == 'image/tiff' ? false : true
```

Comment:
This is implemented because Ryan's comment for not allowing public users to download the TIFF files, see  https://github.com/ucsdlib/horton/issues/111#issuecomment-316482056,.